File: output/ChatGPT/codenet/Python/C++/s747093551.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    int N, M, X, Y;
    cin >> N >> M >> X >> Y;
    
    vector<int> x(N);
    for (int i = 0; i < N; i++) {
        cin >> x[i];
    }
    
    vector<int> y(M);
    for (int i = 0; i < M; i++) {
        cin >> y[i];
    }
    
    if (*max_element(x.begin(), x.end()) < *min_element(y.begin(), y.end()) &&
        max(X, *max_element(x.begin(), x.end())) < min(Y, *min_element(y.begin(), y.end()))) {
        cout << "No War" << endl;
    } else {
        cout << "War" << endl;
    }
    
    return 0;
}

File: output/ChatGPT/codenet/Python/Java/s747093551.java
int N, M, X, Y;
Scanner scanner = new Scanner(System.in);
N = scanner.nextInt();
M = scanner.nextInt();
X = scanner.nextInt();
Y = scanner.nextInt();

int[] x = new int[N];
for (int i = 0; i < N; i++) {
    x[i] = scanner.nextInt();
}

int[] y = new int[M];
for (int i = 0; i < M; i++) {
    y[i] = scanner.nextInt();
}

if (Math.max(Math.max(x), X) < Math.min(Math.min(y), Y)) {
    System.out.println("No War");
} else {
    System.out.println("War");
}